tools/Dashboard/DevHome.Dashboard/Controls/WidgetControl.xaml.cs

```diff
@@ -213,8 +220,13 @@ private void OnCustomizeWidgetClick(object sender, RoutedEventArgs e)
         }
     }
 
-    private void OnActualThemeChanged(FrameworkElement sender, object args)
+    private async void OnActualThemeChanged(FrameworkElement sender, object args)
+    {
+        WidgetHeaderIcon.Fill = await WidgetIconCache.GetBrushForWidgetIcon(WidgetSource.WidgetDefinition, ActualTheme);
+    }
+
+    private async void UpdateWidgetHeaderIconFill()

```

Comment:
Since this is not an event handler, consider adding Async suffix and return Task.
> The name of an async method, by convention, ends with an "Async" suffix.
> The return type is one of the following types:
> - Task\<TResult\> if your method has a return statement in which the operand has type TResult.
> - Task if your method has no return statement or has a return statement with no operand.
> - void if you're writing an async event handler.
> - Any other type that has a GetAwaiter method.

Reference: [Task asynchronous programming model](https://learn.microsoft.com/en-us/dotnet/csharp/asynchronous-programming/task-asynchronous-programming-model)